### ISSUE TITLE
Deprecate the Blender "Better" Collada exporter in favor of glTF2, ESCN and FBX

### ISSUE THREAD

**Issue by Calinou** [godotengine/godot-proposals#2130]
### Describe the project you are working on

The Godot editor :slightly_smiling_face: 

### Describe the problem or limitation you are having in your project

The ["Better" Collada exporter](https://github.com/godotengine/collada-exporter) Blender add-on is clearly lacking maintenance, and it seems no one in the core team has time to dedicate to it. Its latest official version doesn't support Blender 2.8, so most users have resorted to using forks nowadays.

At the same time, [formats like glTF 2.0 have become more popular](https://godotengine.org/article/we-should-all-use-gltf-20-export-3d-assets-game-engines). Even [FBX support has been implemented and rewritten](https://godotengine.org/article/fbx-importer-rewritten-for-godot-3-2-4) since the "Better" Collada exporter was introduced in 2014. The [ESCN exporter](https://github.com/godotengine/godot-blender-exporter) has also been released in that time frame.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Deprecate the "Better" Collada exporter Blender add-on and formally announce it as being unmaintained.

***Note:** This proposal does not suggest removing Godot's Collada importer.* I think it's important to still be able to import Collada scenes, especially as people may be porting older projects to Godot 4.0.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

N/A (see above)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

No.

### Is there a reason why this should be core and not an add-on in the asset library?

This proposal is about taking a decision.

**Comment by akien-mga** [comment]
I still it's still a useful addon, and some of the work from thirdparty forks to support recent Blender versions could be merged upstream. We don't have anyone to maintain it in the core team but we could add new maintainers if there are community contributors who want to pick up the torch for this plugin.

If you're interested, feel free to make it known here (there might have been candidates already in issues/PRs on the collada-exporter repo but I have to admit that I wasn't able to follow discussions there for a long time).

Still, I agree that it should not be recommended in its current state, so we can remove it from the download page.

**Comment by YuriSizov** [comment]
Should we now deprecate and remove from the download page ESCN plugin? As mentioned, it is not really maintained, and would be completely obsolete in Godot 4 with blend file support.

**Comment by Calinou** [comment]
> Should we now deprecate and remove from the download page ESCN plugin? As mentioned, it is not really maintained, and would be completely obsolete in Godot 4 with blend file support.

Indeed, I think we can also consider the ESCN Blender exporter to be deprecated for 4.0 as well.

**Comment by aaronfranke** [comment]
Collada support was discussed in the maintainers chat a few days before I posted this comment. The TL;DR of the discussion was that we are fine with keeping Collada support as it is now, there are some people using it and it's mostly working fine for them. Even though it's not actively maintained and there are better alternatives, there's not really a strong reason to drop it or mark it as deprecated, at least for the near future.

**Comment by YuriSizov** [comment]
To be clear, we aren't going to mark it as deprecated just yet, because there is a potential for it being worked on in the foreseeable future, and it's important for certain gamedev markets. If this does no happen, then it is as good as dead, because there are just too many issues with the Collada support for anyone to rely on this as a feature.

**Comment by fire** [comment]
Deprecation talks are noted in Blender's developer community https://devtalk.blender.org/t/moving-collada-i-o-to-legacy-status/34621/13

**Comment by Shadowblitz16** [comment]
This is dumb dae is good for if you want to separate mesh, armature, textures, and materials.

I just found out godot supports dae and I am going to use it.

**Comment by aaronfranke** [comment]
Blender 5.0 dropped support for exporting Collada .dae files, so users who wish to use this format with Blender need to use an older version of Blender (4.x).

**Comment by fire** [comment]
Sidenote, I am keeping ESCN importing alive.
